### ISSUE TITLE
Regenerate the SHOULDERS.md file

### ISSUE THREAD

**Issue by arschles** [gomods/athens#441]
@markbates made a [new tool](https://github.com/gobuffalo/shoulders) to generate `SHOULDERS.md` files (thanks Mark!), so let's use it to regenerate ours!

**Comment by mydiemho** [comment]
I'm trying to work on this but have some questions:

1. I have downloaded https://github.com/gobuffalo/shoulders
1. when I run `shoulders` in the athens directory, I keep getting error for `cannot find package <XYZ>`.  
1. Each time I would run `go get -u package <XYZ>` and there would be error for another missing package

Is there a command that I could run to get all dependencies at once?

**Comment by robjloranger** [comment]
I get the same issue with up to date dependencies

**Comment by robjloranger** [comment]
@mydiemho see https://github.com/gobuffalo/shoulders/issues/2#issuecomment-412120050
Try updating `shoulders` again and see if the fix works.

**Comment by mydiemho** [comment]
should I also add shoulders to the vendor directory?

**Comment by robjloranger** [comment]
No, let's leave it as a tool not a dependency. Then we can add it as part of the CI to check for changes.
